lib/util.js

```diff
@@ -92,10 +99,23 @@ function escapeStyleCode(code) {
 /**
  * @param {string | string[]} format
  * @param {string} text
+ * @param {object} [options={}]
+ * @param {boolean} [options.validateStream=true] - Whether to validate the stream.
+ * @param {Stream} [options.stream=process.stdout] - The output stream to write the styled text to.
  * @returns {string}
  */
-function styleText(format, text) {
+function styleText(format, text, { validateStream = true, stream = process.stdout } = {}) {
   validateString(text, 'text');
```

Comment:
```suggestion
  validateString(text, 'text');
  validateBoolean(validateStream, 'options.validateStream');
```

Reply:
Done in [cc08b60](https://github.com/nodejs/node/pull/54389/commits/cc08b60053766f5224d2aa95fdc27135e1933412)